yb-voyager/cmd/analyzeSchema.go

```diff
@@ -433,6 +402,52 @@ func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	}
 }
 
+// Checks Whether there is a GIN index
+/*
+Following type of SQL queries are being taken care of by this function -
+	1. CREATE INDEX index_name ON table_name USING gin(column1, column2 ...)
+	2. CREATE INDEX index_name ON table_name USING gin(column1 [ASC/DESC/HASH])
+*/
+func checkGinVariations(createIndexNode *pg_query.Node_IndexStmt, sqlStmtInfo sqlInfo, fpath string) {
+	indexName := createIndexNode.IndexStmt.GetIdxname()
+	relName := createIndexNode.IndexStmt.GetRelation()
+	schemaName := relName.GetSchemaname()
+	tableName := relName.GetRelname()
+	fullyQualifiedName := lo.Ternary(schemaName != "", schemaName+"."+tableName, tableName)
+	displayObjectName := fmt.Sprintf("%s ON %s", indexName, fullyQualifiedName)
+	if createIndexNode.IndexStmt.GetAccessMethod() != "gin" { // its always in lower
+		return
+	} else {
+		summaryMap["INDEX"].details[GIN_INDEX_DETAILS] = true
+	}
+	/*
+		e.g. CREATE INDEX idx_name ON public.test USING gin (data, data2);
+		stmt:{index_stmt:{idxname:"idx_name" relation:{schemaname:"public" relname:"test" inh:true relpersistence:"p"
+		location:125} access_method:"gin" index_params:{index_elem:{name:"data" ordering:SORTBY_DEFAULT nulls_ordering:SORTBY_NULLS_DEFAULT}}
+		index_params:{index_elem:{name:"data2" ordering:SORTBY_DEFAULT nulls_ordering:SORTBY_NULLS_DEFAULT}}}} stmt_location:81 stmt_len:81
```

Comment:
nit: can we make the json readable, put some indentations and new lines.

Reply:
It will unnecessarily take up a lot of lines. The way to reduce that is just to put relevant parts from json for that issue. Will maybe take it up separately.